src/app.py

```diff
@@ -1,31 +1,31 @@
 """
     Leetcode bot application for the TrainingWheels Discord
 """
+
+import argparse
 from dotenv import dotenv_values
 
 from utils.database_util import DatabaseUtil
-#from utils.discord_util import DiscordUtil
+from utils.discord_util import DiscordUtil
 from utils.leetcode_util import LeetcodeUtil
-#from utils.standalone_util import StandaloneUtil
-
+from utils.standalone_util import StandaloneUtil
 
 
 
 if __name__ == "__main__":
     # Program entry point. Parse arguments and launch relevant initialization routines.
 
-
     config = dotenv_values(".process.env")
 
     #Note: this is currently a dummy token. Update with official discord bot token once registered.
     DISCORD_AUTH_TOKEN = config.get("DISCORD_AUTH_TOKEN")
 
     # Example of how to pull all leetcode questions and enter into the database
-    # This will eventually be part of our app initialization routine
+    # # This will eventually be part of our app initialization routine
     new_questions = LeetcodeUtil.api_questions_loadall()
     database = DatabaseUtil(config.get("DATABASE_NAME"))
-    count = database.table_leetcodequestion_insert_many(new_questions)
-    print(f"Inserted {count} new questions into the database")
+    COUNT = database.table_leetcodequestion_insert_many(new_questions)
+    print(f"Inserted {COUNT} new questions into the database")
```

Comment:
My local linter also complained about `COUNT` needing to be constant because it wasn't referenced elsewhere in the code. I disagree with that in principle because the value of the variable is determined at runtime and not statically ahead of time. In my opinion it's a poor linter rule, and I think I disabled it in the pylintrc file in this solution, but I'm okay with us using uppercase `COUNT` here.

Reply:
I found that weird, too! Leaving it as is for now, but happy to switch to lowercase count if decided.